conf/machine/include/tune-riscv.inc

```diff
@@ -17,3 +17,7 @@ TUNE_ARCH_tune-riscv32 = "riscv32"
 TUNE_PKGARCH_tune-riscv32 = "riscv32"
 PACKAGE_EXTRA_ARCHS_tune-riscv32 = "riscv32"
 
+# QEMU usermode fails with invalid instruction error (YOCTO: #10304)
+MACHINE_FEATURES_BACKFILL_CONSIDERED_append_riscv64 = "${@bb.utils.contains('TUNE_FEATURES', 'riscv64', ' qemu-usermode', '', d)}"
+MACHINE_FEATURES_BACKFILL_CONSIDERED_append_riscv32 = "${@bb.utils.contains('TUNE_FEATURES', 'riscv32', ' qemu-usermode', '', d)}"
```

Comment:
@pino-kim please add this change to `arch-riscv.inc` and there is no need to have overrides since this will be added to risc-v machines alone

```
MACHINE_FEATURES_BACKFILL_CONSIDERED_append = " ${@bb.utils.contains('TUNE_FEATURES', 'riscv32', ' qemu-usermode', '', d)}"
```
I wonder if we need this change for riscv64

Reply:
@kjraj thanks to reply soon.
I attach my change to ```arch-riscv.inc```.
Test both arch( riscv64, riscv32) again and re PR again.  
It look look better I think.

>+ #QEMU usermode fails with invalid instruction error (YOCTO: #10304)
> + MACHINE_FEATURES_BACKFILL_CONSIDERED_append_riscv64 = "${@bb.utils.contains('TUNE_FEATURES', '${DEFAULTTUNE}', ' qemu-usermode', '', d)}"

Thanks